src/lib.rs

```diff
@@ -1,6 +1,6 @@
 pub mod poll;
 pub mod socket;
-mod wasi_poll;
+pub mod wasi_poll;
```

Comment:
wasi_poll are marked as public here because other libs are using wasi_poll directly.
once I modify those libs, we can make it unpublic again.